marketplace_standard_app_api/models/object_storage.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from typing import List, NewType, Optional
+
+from pydantic import UUID4, BaseModel
+
+DatasetId = NewType("DatasetId", UUID4)
+
+
+class DatasetCreateResponse(BaseModel):
+    last_modified: datetime
+
+
+class DatasetModel(BaseModel):
+    id: DatasetId
+    hash: Optional[str]
+    bytes: Optional[int]
+    content_type: Optional[str]
+    last_modified: Optional[datetime]
+
+
+class DatasetListResponse(BaseModel):
+    items: List[DatasetModel]
```

Comment:
Please note that the SWIFT response is an array: https://docs.openstack.org/api-ref/object-store/index.html#show-container-details-and-list-objects

That's a bit annoying since we went with this approach for the transformation and globalSearch "list-responses". I do not like that we are violating the API here, but if we change it here, we might want to change this for the other responses as well.